R/R-to-variable.R

```diff
@@ -61,14 +61,32 @@ setMethod("toVariable", "logical", function (x, ...) {
     return(NAToCategory(out, useNA="always"))
 })
 
-categoriesFromLevels <- function (x) {
-    if (anyDuplicated(x)) {
+#' Convert a factor's levels into Crunch categories. 
+#' 
+#' Crunch categorical variables have slightly richer metadata than R's
+#' factor variables. This function generates a list of category data from
+#' a factor's levels which can then be further manipulated in R before being
+#' imported into Crunch.
+#'
+#' @param levels A character vector containing the levels of a factor. Usually
```

Comment:
There's something odd about naming the argument the same as the function you expect someone to call to get the values for it since in the function scope, `levels` is now an object and not that function. But it seems to work, and I don't have a better suggestion for the name of the variable.